app/src/main/java/net/squanchy/analytics/FirstStartUserPropertiesPersister.kt

```diff
@@ -0,0 +1,19 @@
+package net.squanchy.analytics
+
+import android.content.SharedPreferences
+
+internal class FirstStartUserPropertiesPersister(private val preferences: SharedPreferences) {
```

Comment:
I'd rename it to `FirstStartDetector` and leave it as a more general-purpose tool